spec/unit/installer/xcode/target_validator_spec.rb

```diff
@@ -46,6 +40,7 @@ def create_validator(sandbox, podfile, lockfile, integrate_targets = false)
             config.repos_dir = fixture_path + 'spec-repos'
             podfile = Pod::Podfile.new do
               platform :ios, '8.0'
+              install! 'cocoapods', :integrate_targets => false
```

Comment:
very nice does this speed up the tests?

Reply:
before the tests were injecting the option into the installation options to control this, so no I actually had to add it just to get it to work correctly

Reply:
ah